inst/sympref.m

```diff
@@ -306,6 +306,9 @@
             msg = 'Choosing the default [autodetect] communication mechanism';
           case 'pytave'
             msg = 'Forcing the PyTave communication mechanism';
+            pyexec(strjoin({'import sympy as sp',
+                            'from sympy import __version__ as spver', 
+                            'from sympy import *'}, newl));
```

Comment:
anyway, I can't test becuase `newl` is not defined.  Its kinda rude to ask for review when there are trivial problems.